src/components/KYCWall/BaseKYCWall.js

```diff
@@ -100,7 +100,8 @@ class KYCWall extends React.Component {
             this.setState({shouldShowAddPaymentMenu: false});
             return;
         }
-        this.setState({transferBalanceButton: event.nativeEvent.target});
+        const targetElement = this.anchorRef.current || event.nativeEvent.target; // safety check - use target from event if anchorRef is null
```

Comment:
Let's add the comment above the code 

```suggestion
        // Use event target as fallback if anchorRef is null for safety
        const targetElement = this.anchorRef.current || event.nativeEvent.target;
```